Tests/SwiftDocCTests/Test Resources/Inheritance.symbols.json

```diff
@@ -263,7 +263,7 @@
                 "character" : 26
               }
             },
-            "text" : "@returns The index"
+            "text" : "@Comment returns a comment"
```

Comment:
These test changes still don't make sense to me. Why do we need to remove the doxygen commands here?

Reply:
Hi Ethan, i am not removing the doxygen comment, instead of "@returns" (unknown directive) i am specifying "@Comment" a known directive

Reply:
So the `@returns` is a valid Doxygen command (https://www.doxygen.nl/manual/commands.html#cmdreturns). By replacing `@returns` with `@Comment` we're effectively replacing this Doxygen comment with a DocC Documentation Markup one.

Since we expect DocC to process existing doxygen comments I think it's important for us to have some text fixtures that include Doxygen commands like `@returns`.

If there's an existing test that's starting to fail with this change because `@returns` is no longer causing a diagnostic, I think we should change the test itself instead of changing this test fixture.